cmd/notary/integration_test.go

```diff
@@ -2269,7 +2293,7 @@ func TestClientKeyImport(t *testing.T) {
 	// if there is hardware available, root will only be on hardware, and not
 	// on disk
 	assertNumKeys(t, tempDir, 2, 4, !rootOnHardware())
-	_, err = os.Open(filepath.Join(tempDir, "private", notary.NonRootKeysSubdir, newKeyID+".key"))
+	_, err = os.Open(filepath.Join(tempDir, notary.PrivDir, notary.NonRootKeysSubdir, newKeyID+".key"))
```

Comment:
thanks for catching that, since it was checking that the file didn't exist I missed that one 😄  will update it with the next batch of changes